src/main/java/seedu/address/model/task/Task.java

```diff
@@ -11,50 +17,75 @@
  */
 public class Task {
 
+    public static final String DATE_FORMAT = "d/M/yyyy";
+
+    public static final DateTimeFormatter DATE_FORMAT_FOR_PRINT = DateTimeFormatter.ofPattern("d/M/yyyy");
+    public static final DateTimeFormatter DATE_FORMATTER_FOR_USER_INPUT = DateTimeFormatter.ofPattern(DATE_FORMAT);
```

Comment:
hmmm what's going on here?

Reply:
Used for date field parser and JSON parser for AddTask